flake.nix

```diff
@@ -30,18 +27,18 @@
     };
   };
 
-  outputs = { self, hydra-provisioner, nixpkgs, utils, bitte-cli, nixpkgs-2105, ... }@inputs:
+  outputs = { self, hydra, hydra-provisioner, nixpkgs, utils, bitte-cli, ... }@inputs:
   let
     lib = import ./lib { inherit (nixpkgs) lib; };
   in utils.lib.simpleFlake rec {
     inherit lib nixpkgs;
 
     systems = [ "x86_64-linux" ];
 
+    preOverlays = [ bitte-cli hydra-provisioner hydra ];
```

Comment:
does this need to be exported?

Reply:
https://github.com/kreisys/flake-utils/blob/master/simpleFlake.nix#L16-L19

`preOverlays` is just a compatibility alias for `overlays`. The singular `overlay` is just a convenience and it gets appended to `overlays` (which, in turn, default to `[]`). In the end, the `overlay` that's exposed in the attrset that is returned by the `simpleFlake` function (which happens to have an argument of the same name), is a single one, composed of all the overlays that had been passed into `simpleFlake` using the `composeManyExtensions` library function from `nixpkgs.lib`. Y'all need to give me more credit 🙃

Reply:
thanks for heads up :+1: